app/screens/more_channels/more_channels.js

```diff
@@ -201,13 +202,20 @@ class MoreChannels extends PureComponent {
     };
 
     onSelectChannel = async (id) => {
+        const {actions, currentTeamId, currentUserId} = this.props;
+        const {channels} = this.state;
+
         this.emitCanCreateChannel(false);
         this.setState({adding: true});
-        await this.props.actions.joinChannel(
-            this.props.currentUserId,
-            this.props.currentTeamId,
-            id);
-        await this.props.actions.handleSelectChannel(id);
+        await actions.joinChannel(currentUserId, currentTeamId, id);
```

Comment:
yup working on this right now https://mattermost.atlassian.net/browse/RN-239, so lets merge  this one